NEWS.md

```diff
@@ -1,5 +1,8 @@
 # spatialsample (development version)
 
+* `block_cv()` is a new function for performing spatial block cross validation.
```

Comment:
```suggestion
* `spatial_block_cv()` is a new function for performing spatial block cross validation.
```

Reply:
I wrapped all of these into a commit locally -- wanted to `grep` through to make sure I caught all of them

Reply:
I wrapped all of these into a commit locally -- wanted to `grep` through to make sure I caught all of them